extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/i18n/sonos_de.properties

```diff
@@ -0,0 +1,130 @@
+thing-type.config.sonos.zoneplayer.udn.description = Der UDN zur Identifizierung des Sonos Ger�tes.
+thing-type.config.sonos.zoneplayer.notificationTimeout.label = Timeout f�r Benachrichtigungen
+thing-type.config.sonos.zoneplayer.notificationTimeout.description = Timeout f�r Benachrichtigungen (in Sekunden).
+thing-type.config.sonos.zoneplayer.refresh.label = Abfrageintervall
+thing-type.config.sonos.zoneplayer.refresh.description = Intervall zur Abfrage des Sonos Ger�tes (in Sekunden).
+
+# channel types
+channel-type.sonos.add.label = Hinzuf�gen
+channel-type.sonos.add.description = F�gt den aktuellen Sonos Speaker zur aktuellen Gruppe hinzu.
+channel-type.sonos.alarm.label = Alarm aktivieren
+channel-type.sonos.alarm.description = Erm�glicht die Aktivierung des Alarms. Alarm Einstellung m�ssen in der Sonos Controller App erfolgen.
+channel-type.sonos.alarmproperties.label = Alarm Einstellungen
+channel-type.sonos.alarmproperties.description = Zeigt die Einstellungen des aktuellen Alarms an.
+channel-type.sonos.alarmrunning.label = Alarm aktiviert?
+channel-type.sonos.alarmrunning.description = Zeigt an, ob der Alarm aktiviert ist.
+channel-type.sonos.clearqueue.label = Wiedergabeliste leeren
+channel-type.sonos.clearqueue.description = Leert die aktuelle Wiedergabeliste.
+channel-type.sonos.control.label = Fernbedienung
+channel-type.sonos.control.description = Erm�glicht die Steuerung der Wiedergabe von Medien auf dem Sonos Speaker (z.B. Play, Pause).
+channel-type.sonos.coordinator.label = Master
+channel-type.sonos.coordinator.description = Zeigt den UDN des Masters der aktuellen Gruppe an.
+channel-type.sonos.currentalbum.label = Album
+channel-type.sonos.currentalbum.description = Zeigt das Album des aktuellen St�cks an.
+channel-type.sonos.currentalbumart.label = Coverbild
+channel-type.sonos.currentalbumart.description = Zeigt das Coverbild des aktuellen St�cks an.
+channel-type.sonos.currentalbumarturl.label = Coverbild URL
+channel-type.sonos.currentalbumarturl.description = Zeigt die URL zum Coverbild des aktuellen St�cks an.
+channel-type.sonos.currentartist.label = K�nstler
+channel-type.sonos.currentartist.description = Zeigt den K�nstler des aktuellen St�cks an.
+channel-type.sonos.currenttitle.label = Titel
+channel-type.sonos.currenttitle.description = Zeigt den Titel des aktuellen St�cks an.
+channel-type.sonos.currenttrack.label = Track
+channel-type.sonos.currenttrack.description = Zeigt den Track des aktuellen St�cks an.
+channel-type.sonos.currenttrackuri.label = Track URI
+channel-type.sonos.currenttrackuri.description = Zeigt die URI des aktuellen Tracks an.
+channel-type.sonos.currenttransporturi.label = AV Transport URI
+channel-type.sonos.currenttransporturi.description = Zeigt die URI des aktuellen AV Transportes an.
+channel-type.sonos.favorite.label = Favorit
+channel-type.sonos.favorite.description = Erm�glicht das Abspielen des Favoriten. Einstellung von Favoriten m�ssen in der Sonos Controller App erfolgen.
```

Comment:
Is this only one favorite? Or should it be "... eines Favoriten."?

Reply:
Yes, it is only one favorite. I will change ist to '... des eingestellten Favoriten.'.